data/projects.js

```diff
@@ -428,8 +428,8 @@ coala should support generating metrics for your code.",
 		women and students. ",
 		"requirements" : [
 				"The applicant should be be familiar with bear creation and XML parsing \
-				using xpath or a similar description language",
-				"The applicant reasonable familiar with the coala codebase and the workings of REST APIs, \
+				using xpath or a similar query language",
+				"The applicant should be reasonable familiar with the coala codebase and the workings of REST APIs, \
 				including authentication."
```

Comment:
Can you add a contribution to either to a relevant coala module or to the Open Roberta platform or both. Having this requirement saves a lot of time during the bonding period and hence we can focus on mocking up the project better during that time.